web/src/contexts/intercomContext.ts

```diff
@@ -0,0 +1,12 @@
+import { OperatingPhaseId } from "@businessnjgovnavigator/shared/operatingPhase";
+import { createContext } from "react";
+
+export interface IntercomContextType {
+  operatingPhaseId: OperatingPhaseId | undefined;
```

Comment:
I think that this context doesn't actually need to include the operatingPhaseId. Consumers of this context are only ever using the setter.